segments/kubecontext/kubecontext.p9k

```diff
@@ -36,6 +36,11 @@ prompt_kubecontext() {
       cur_namespace="default"
     fi
 
+    # Extract cluster name and namespace from AWS ARN, if enabled.
+    if [[ "$P9K_KUBECONTEXT_STRIPEKS" == true ]]; then
+      cur_ctx="$(echo $cur_ctx | cut -d':' -f 6 | sed 's/cluster\/*//')"
```

Comment:
This should do the same thing:
```suggestion
      cur_ctx="${${(s.:.)cur_ctx}[6]#cluster/}"
```

Reply:
Please check before use :)

Reply:
Or this: `"${${cur_ctx##*\:}/cluster\//}`. 😄

Reply:
I hope there is no `:` in the last part. Because then all suggestions fail :laughing:
If there are not additional `/`s then you could even go for `${cur_ctx:t}`